x-pack/plugins/cloud_security_posture/public/pages/rules/translations.ts

```diff
@@ -84,3 +84,8 @@ export const OVERVIEW = i18n.translate('xpack.csp.rules.ruleFlyout.tabs.overview
 export const REMEDIATION = i18n.translate('xpack.csp.rules.ruleFlyout.tabs.remediationTabLabel', {
   defaultMessage: 'Remediation',
 });
+
+export const DATA_UPDATE_INFO = i18n.translate('xpack.csp.rules.dataUpdateInfoCallout', {
+  defaultMessage:
+    'Please note, any changes to your benchmark rules will take effect the next time your resources are evaluated again. This can take up to ~5 hours',
+});
```

Comment:
i think we should drop the word `again`. 

"next time resources are evaluated" seems better?

Reply:
++ get rid of it!